ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -30,6 +30,7 @@ ColumnLayout {
     id: root
 
     required property var collectiblesModel
+    required property var controller
```

Comment:
Could be strong typed. We know exactly what controller we need here.

Reply:
Yeah... but I'm usually against it since in general, e.g. in StoryBook you might want to pass something else :)